content/docs/preferences.md

```diff
@@ -168,8 +168,9 @@ This affects the display of the main Language menu, and also affects the per-lan
     * `normal` sets the tab behavior for plain text
     * other selections will choose which syntax-language the tab settings are being changed
     * `☐ Use default value`: not visible on the `[Default]` selection.  For other languages, will use the values from the `[Default]` selection for that particular language
-    * `Tab size : ___`: sets the width of the tab character, or the number of spaces to use instead of a tab
-    * `☐ Replace by space`: when set, hitting TAB will insert that number of spaces; when not set, TAB will insert the literal ASCII TAB character
+    * `Tab size : ___`: sets the width of the tab stop.  If the text uses literal TAB characters (`Replace by space` is turned off), there will be one TAB character in the file, but Notepad++ will _display_ that character as a gap with a width of up to that many space characters.  If the text is replacing TABs with spaces, enough literal spaces will be inserted into the file so the next character will start at the beginning of the next tab stop.
```

Comment:
is turned off), there will be one TAB character in the file, but
->
is turned off), a press of the Tab key will insert one TAB character into the file, but
??